infrastructure-playbooks/cephadm-adopt.yml

```diff
@@ -360,14 +360,22 @@
       delegate_to: '{{ groups[mon_group_name][0] }}'
       when: is_hci | bool
 
+    - name: set autotune_memory_target_ratio
+      command: "{{ ceph_cmd }} config set mgr mgr/cephadm/autotune_memory_target_ratio {{ '0.2' if is_hci | bool else '0.7' }}"
```

Comment:
I'm asking out of ignorance (not very familiar with hci setup and requirements) why wouldn't it make sense?

Reply:
No wait, this task makes sense to me (a lot!) [1].
In fresh cephadm hci based deployments we pass the `mgr/cephadm/autotune_memory_target_ratio` key and set that value (more info and description in [1]).
I didn't see you already added that task because of [2] via `https://github.com/ceph/ceph-ansible/commit/41d62596fcbe01c83c97bfa28ecbf7ee00b9d072`, so we're good here! :D

[1] https://docs.openstack.org/project-deploy-guide/tripleo-docs/latest/features/deployed_ceph.html#scenario-deploy-hyperconverged-ceph
[2] https://bugzilla.redhat.com/show_bug.cgi?id=2028693

Reply:
👍